changelog.md

```diff
@@ -8,11 +8,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 - Handle stdlib semver syntax in the `status` command ([#290](https://github.com/zeppelinos/zos-cli/issues/290))
-<<<<<<< HEAD
 - Better wording from status command when a contract is scheduled to be removed ([#304](https://github.com/zeppelinos/zos-cli/issues/304))
-=======
+- Fail with an explicit error when attempting to create a proxy for an stdlib that was linked but not pushed to the network ([#322](https://github.com/zeppelinos/zos-cli/pull/322))
 - Include default timeout in `--timeout` flag help ([#302](https://github.com/zeppelinos/zos-cli/issues/302))
->>>>>>> Clarify timeout flag and show default value (fixes #302)
```

Comment:
I guess you missed some conflicts here 😝

Reply:
Actually it's the other way around. There are conflicts that were pushed into master :scream:

Reply:
🙀

Reply:
Thanks for fixing them :)